### ISSUE TITLE
other lanuage

### ISSUE THREAD

**Issue by Moonmore** [omine-me/LaughterSegmentation#3]
Hello, thank you very much for your great work, I will try to use your model normally.

I would like to ask if this model works for Chinese. In other words, the model only targets the original voice of the speech, and can be used for the language of the speech.

**Comment by omine-me** [comment]
Thank you for your interest.
It is difficult to answer because I haven't done much evaluation in Chinese, but I think it works in basically any language. This is because I feel that laughter does not vary greatly from language to language. However, there is a possibility that certain characteristics of Chinese might be misinterpreted as laughter.

Most of the laughter used in the study was from English and Japanese speakers, but a variety of speakers' voices were used. The base audio is also mostly English, but uses a variety of video clips.
Some of them may be Chinese.
If you want even more accuracy, you can finetune it. 😊

**Comment by Moonmore** [comment]
Thanks for your replay.

I take the liberty to ask you a few questions.

1. I am doing some research on laughter detection. I would like to ask what makes your model different from other model? The paper seem to be searchable yet.
2. If finetune the model, how should I prepare the data format and how much data is required?

Best regards

**Comment by omine-me** [comment]
Here's a link to the pdf.
[Robust Laughter Segmentation with Automatic Diverse Data Synthesis](https://drive.google.com/file/d/1WqYKGvCC3b-l8zN06ESh8GlcqJyC9hLm/view?usp=sharing)

1. In simple terms, the idea is that by synthesizing the laughter voice and arbitrary audio, a variety of data could be easily created. The model itself isn't that special.
1. You have to prepare laughter audio and arbitrary audio (basically similar to the audio environment you want to infer). Mono 16kz wav is ideal. I don't know how much data is needed for finetuning, but FYI, training from scratch, we used about 15,000 laughter samples.

Hope your research goes well.

**Comment by Moonmore** [comment]
Thanks for your replay again. I will try it with your suggestion.

Best regards and many thanks.

**Comment by Lindsay284** [comment]
> Thanks for your replay again. I will try it with your suggestion.再次感谢您的重播。我会按照你的建议试试。
> 
> Best regards and many thanks.最诚挚的问候和非常感谢。

Hello, may I ask if you have done fine-tuning for Chinese? I'm currently working on a similar task, and if you've already done some work, would you be open to exchanging insights?
